packages/charts/src/charts/bar/utils/bar.utils.ts

```diff
@@ -135,17 +134,24 @@ export const calculateGroupedBars = (
         });
     });
   }
+  const bars = [] as Bar[];
```

Comment:
I do not see any `unit tests` addressing this changes at `calculateGroupedBars` function level. Could You please add test cases for this function - that covers bar ordering and positioning logic ?

Reply:
`calculateGroupedBars` tests are in bar.utils.test.ts file